src/models/converters/balances.rs

```diff
@@ -1,5 +1,5 @@
 use crate::models::backend::balances::Balance as BalanceDto;
-use crate::models::chains::NativeCurrency;
+use crate::models::backend::chains::NativeCurrency;
```

Comment:
nitpick: I would actually consider anything under `crate::models` to be part of this application so I would not put them under `::backend`. As for the client models then it'd make more sense to put them under `crate::models::clients::` wdyt?

This is out of scope for this PR anyways.

Reply:
We for sure change that if you like. In a different PR :) . For now the distinction is:

- `crate::model::backend` for classes in the config and tx services
- `crate::model::service` for classes exposed by this service
- `crate::model::commons` for classes which aren't mapped and are forwarded as is to clients

These names are nice because the have the same length in a monospaced font :P

Reply:
Admittedly, `commons` should be refactored into a folder and not just as a flat `.rs` file